background/constants/networks/networks.ts

```diff
@@ -1,5 +1,72 @@
 import { NetworkInterface } from "./networkTypes"
 
+export const DEFAULT_PELAGUS_NETWORK = {
```

Comment:
No need to do this.

I assume that we can do it like this:
`export const DEFAULT_PELAGUS_NETWORK = QuaiGoldenAgeTestnet`